examples/02_model_collaborative_filtering/multi_vae_deep_dive_2.ipynb

```diff
@@ -0,0 +1,1998 @@
+{
```

Comment:
<p>could you please remove colab's references?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>